### DISCUSSION TITLE
You are marked as spam, and therefore cannot authorize a third party application.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#136385] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![image](https://github.com/user-attachments/assets/986dec66-f656-4a6e-9455-8977acbda604)
Good evening, I need some advice on how to fix this issue. I was excited to get a new computer, so I quickly downloaded a bunch of apps and pulled some code to start learning. But when I tried to pull code using the HTTPS protocol, something unexpected happened. As shown in the picture, I’ve been flagged as spam, so I can’t authorize third-party apps. This means that even though I can log in normally, I can’t pull code via HTTPS. I asked a friend to check if they could see my code, but they couldn't find any information about me at all.
Before publication, I had already submitted a work order on the support page with the number # 2934119

**Comment by LiteBrite82** [comment]
Hi @MingxuCarry 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

That being said, did you see this discussion: [WHY IS MY ACCOUNT FLAGGED AS SPAM?! #60356](https://github.com/orgs/community/discussions/60356), it may or may not shine some light on the issue.
